semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/http/HttpSemanticConventionUtils.java

```diff
@@ -439,13 +455,46 @@ public static URL getNormalizedUrl(String url) throws MalformedURLException {
   public static Optional<Integer> getHttpRequestSize(Event event) {
     String httpRequestSize =
         SpanAttributeUtils.getFirstAvailableStringAttribute(event, REQUEST_SIZE_ATTRIBUTES);
-    return Optional.ofNullable(httpRequestSize).map(s -> Integer.parseInt(s));
+    return Optional.ofNullable(httpRequestSize).map(Integer::parseInt);
   }
 
   public static Optional<Integer> getHttpResponseSize(Event event) {
     String httpResponseSize =
         SpanAttributeUtils.getFirstAvailableStringAttribute(event, RESPONSE_SIZE_ATTRIBUTES);
-    return Optional.ofNullable(httpResponseSize).map(s -> Integer.parseInt(s));
+    return Optional.ofNullable(httpResponseSize).map(Integer::parseInt);
+  }
+
+  public static int getHttpResponseStatusCode(Event event) {
+
+    if (event.getAttributes() == null || event.getAttributes().getAttributeMap() == null) {
+      return 0;
```

Comment:
Here, zero means we couldn't able to derive the status code. Do you think, here as well we can use Optional<Integer> as return type?
cc : @rish691

Reply:
private int getResponseStatusCode(Event event) {
    return Optional.ofNullable(event.getHttp())
        .map(Http::getResponse)
        .map(Response::getStatusCode)
        .orElse(0);
  }
  
  This was the original code to get status code.

Reply:
oh! okey so we were already using `0`.